src/src/Helpers/StringConverter.cpp

```diff
@@ -800,6 +800,47 @@ String tolerantParseStringKeepCase(const String& string, uint8_t indexFind, char
   return parseStringKeepCase(string, indexFind, separator, trimResult);
 }
 
+/*****************************************************************************
+ * handles: 0xXX,text,0xXX," more text ",0xXX starting from index 2 (1-based)
+ ****************************************************************************/
+String parseHexTextString(const String& argument, int index) {
```

Comment:
Didn't we already have something like this?
After all, the numerical parser does also accept HEX strings.